template.yaml

```diff
@@ -877,16 +909,16 @@ Resources:
       KeySchema:
         - AttributeName: "zendeskId"
           KeyType: "HASH"
-      GlobalSecondaryIndexes:
-        - IndexName: athenaQueryIdIndex
-          KeySchema:
-            - AttributeName: "athenaQueryId"
-              KeyType: "HASH"
-          Projection:
-            ProjectionType: "KEYS_ONLY"
-          ProvisionedThroughput:
-            ReadCapacityUnits: 5
-            WriteCapacityUnits: 5
+      # GlobalSecondaryIndexes:
```

Comment:
Did you mean to delete instead of commenting out?

Reply:
Hmm, the hope was that this would get around the inability to update a GSI in a stack but the deployment has failed, so another approach may be required.

Reply:
We'll be dealing with this during the merge to main, may be a case of trial and error to some extent